docs/tutorial/category_theory/calculating_colimits_in_Top.lean

```diff
@@ -24,19 +24,19 @@ def I_1 : pt ⟶ I :=
 { val := λ _, ⟨(1 : ℝ), begin rw [set.right_mem_Icc], norm_num, end⟩,
   property := continuous_const }
 
-def cylinder (X : Top) : Top := limit (pair X I)
+def cylinder (X : Top) : Top := prod X I
 -- To define a map to the cylinder, we give a map to each factor.
 -- `binary_fan.mk` is a helper method for constructing a `cone` over `pair X Y`.
 def cylinder_0 (X : Top) : X ⟶ cylinder X :=
-limit.lift (pair X I) (binary_fan.mk (𝟙 X) (to_pt X ≫ I_0))
+prod.lift (𝟙 X) (to_pt X ≫ I_0)
 def cylinder_1 (X : Top) : X ⟶ cylinder X :=
-limit.lift (pair X I) (binary_fan.mk (𝟙 X) (to_pt X ≫ I_1))
+prod.lift (𝟙 X) (to_pt X ≫ I_1)
 
 -- The mapping cylinder is the colimit of the diagram
 --    X
 --   ↙ ↘
 --  Y   (X x I)
-def mapping_cylinder {X Y : Top} (f : X ⟶ Y) : Top := colimit (span f (cylinder_1 X))
+def mapping_cylinder {X Y : Top} (f : X ⟶ Y) : Top := pushout f (cylinder_1 X)
 
 -- The mapping cone is the colimit of the diagram
```

Comment:
`s/colimit/pushout/`?

Reply:
No, this one is genuinely a colimit. See also the next diagram. I've made the comment clearer.